cobigen-plugins/cobigen-javaplugin-parent/cobigen-javaplugin/src/main/java/com/devonfw/cobigen/javaplugin/merger/JavaMerger.java

```diff
@@ -88,6 +89,19 @@ public String merge(File base, String patch, String targetCharset) throws MergeE
       throw new MergeException(base, "The syntax of the base file is invalid. Error in line: " + e.getLine()
           + " / column: " + e.getColumn() + ": " + e.getMessage(), e);
     }
+    if (patch == null) {
+      JavaClass parsedAnnotationClass;
+      LocalDate localDate = LocalDate.now();
+      String annotation = "@Generated(value={\"com.devon.CobiGen\"}, date = \"" + localDate + "\")"
+          + "public class Anno{}";
```

Comment:
public class Anno{} ?

Reply:
getFistJavaClass in JavaParserUtil reads the content of class. so far this is the only way I figured out to extract generated annotation which also comply with project standards . If you have something else in mind, Would be happy to know

Reply:
I would try to create a DefaultJavaAnnotation using a ModifyableJavaClass instead.